InnoTopicWebsite/src/app/skills/work-experience.component.html

```diff
@@ -35,72 +35,74 @@ <h1 class="section-title">Overview</h1>
   <!-- FIXME: extract languages component -->
   <h2 class="section-title">Languages</h2>
   <!-- FIXME: 2..4 columns -->
-<!--  <app-flags flags="de at lu pl gb us pt es es-ct fr" />-->
+  <!--  <app-flags flags="de at lu pl gb us pt es es-ct fr" />-->
   <!-- FIXME: extract component for flag + label-->
-  <div class="language">
-    <div class="app-flags"><app-flags flags="pl" /></div>
-    <label>Polish</label>
-    <span>Native</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="gb us" /></div>
-    <label>English</label>
-    <span>Full professional proficiency</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="es mx" /></div> <!-- TODO: argentina -->
-    <label>Spanish</label>
-    <span>Full professional proficiency</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="de at" /></div> <!-- TODO CH-->
-    <label>German</label>
-    <span>Full professional proficiency</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="es-ct" /></div>
-    <label>Català</label>
-    <span>Intermediate</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="pt br" /></div>
-    <label>Portuguese</label>
-    <span>Intermediate</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="fr be" /></div>
-    <label>French</label>
-    <span>Beginner</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="it" /></div>
-    <label>Italian</label>
-    <span>Beginner</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="nl be" /></div>
-    <label>Dutch</label>
-    <span>Beginner</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="ru" /></div>
-    <label>Russian</label>
-    <span>Beginner</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="cn" /></div>
-    <label>Mandarin Chinese</label>
-    <span>Beginner</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="ma"/></div>
-    <label>Arabic</label>
-    <span>Beginner</span>
-  </div>
-  <div class="language">
-    <div class="app-flags"><app-flags flags="in"/></div>
-    <label>Hindi</label>
-    <span>Beginner</span>
+  <div class="languages-grid">
```

Comment:
let's move this to separate component, and make it declarative in this format in typescript:

{
  English: {
    flags: "gb us",
    level: "Full professional proficiency"

Reply:
@karol-depka I will create a separate task for this. Thanks!

Reply:
@karol-depka I've created a new task for this. https://github.com/InnoTopic/InnoTopic_Website/issues/80